### ISSUE TITLE
Exception on paste with empty clipboard

### ISSUE THREAD

**Issue by alensiljak** [gaphor/gaphor#2737]
### Describe the bug
Trying to copy an element. Ctrl+Shift+V caused an exception. I guess the initial Ctrl+C did not really work, since the error message states that the clipboard is empty.

### To Reproduce
1. Press Ctrl+V on a diagram (without first copying an element)

If possible, add the error report from `≡` (top-right menu) → `Tools` → `Error Reports`.
```
Gaphor version:         2.21.0
Operating System:       Windows (10)
Display:                GdkWin32Display
Python version:         3.11.5
GTK version:            4.12.0
Adwaita version:        1.3.1
GtkSourceView version:  5.8.0
Cairo version:          1.17.8
Pango version:          1.50.14
PyGObject version:      3.44.1
Pycairo version:        1.24.0
pygit2/libgit2 version: 1.12.2  / 1.6.4


Errors:

Time since application startup: 0:10:48
|Traceback (most recent call last):
|  File "C:\Program Files\Gaphor\gaphor\ui\copyservice.pyc", line 74, in on_paste
|gi.repository.GLib.GError: g-io-error-quark: Cannot read from empty clipboard. (1)
```
### Expected behavior
Probably nothing in this case. I suspect the Copy operation did not work but pasting with an empty clipboard should not cause an exception.

### OS
- [x] Windows

### Version
Version of Gaphor: 2.21.0

**Comment by amolenaar** [comment]
Thanks @alensiljak 

This is a bit unfortunate. I can reproduce it with similar outcome:

```
Traceback (most recent call last):
  File "/home/arjan/Development/gaphor/gaphor/ui/copyservice.py", line 74, in on_paste
    copy_buffer = self.clipboard.read_value_finish(result)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-io-error-quark: No compatible formats to transfer clipboard contents. (15)
```

@dependabot Add @alensiljak for bugs.

**Comment by danyeaw** [comment]
@all-contributors please add @alensiljak for bug

**Comment by danyeaw** [comment]
Fixed by #2822. Thanks for the bug report @alensiljak, version 2.22.1 will be out soon with this.
